src/components/interior-left-nav/interior-left-nav.js

```diff
@@ -161,10 +161,13 @@ class InteriorLeftNav extends mixin(createComponent, initComponentBySearch) {
     });
 
     if (this.element.dispatchEvent(eventStart)) {
+      const svgTitle = this.element
+        .querySelector(this.options.selectorLeftNavArrow)
+        .querySelector('title');
       if (!collapsed) {
         this.element.dataset.collapsed = true;
         this.element.classList.add(this.options.classLeftNavCollapsing);
-
+        svgTitle.innerHTML = 'Expand nav pane';
```

Comment:
Two things here and https://github.com/carbon-design-system/carbon-components/pull/319/files#diff-2e7e5be2ee91d6de1c6604f6fa89bbf5R186:

1. I’d suggest `.textContent` instead of `.innerHTML` - The latter triggers HTML parser
2. i18n - I’d suggest putting the text to static `options.expandTitle` property and then make this line refer to `this.options.expandTitle` (Note that all static `options` work as default values for instance `options`)